### DISCUSSION TITLE
Best Way To Align Components in A Group of Input Components?

### DISCUSSION THREAD

**Discussion by elibroftw** [mantinedev/mantine#1217] [Q&A] [answered]
What's the best way to align non label components with label components? Should I just use Space and tune the height or is there a more elegant way?
![image](https://user-images.githubusercontent.com/21298211/162592686-5c9983d2-9b78-4f8d-87b2-ed4bb1165bf3.png)

**Comment by rtivital** [comment, marked as answer]
`display:flex; align-items: flex-end;`

**Reply by ansonkao** [reply]
This doesn't work if the input can have validation errors. In that case, @elibroftw's original suggestion is best
